{"name": "test_private_repository_invalid_token", "status": "failed", "statusDetails": {"message": "AssertionError: Unauthorized access should fail (GitHub hides private repos)\nassert 401 == 404\n +  where 401 = <Response [401]>.status_code", "trace": "def test_private_repository_invalid_token():\n        \"\"\"Step 4: Verify that an invalid token cannot access a private repository.\"\"\"\n        print(\"\\nðŸ”¹ Running: test_private_repository_invalid_token\")\n        invalid_headers = {\n            \"Authorization\": \"Bearer invalid-token\",\n            \"Accept\": \"application/vnd.github.v3+json\"\n        }\n        response = requests.get(f\"{GITHUB_API_URL}/repos/{OWNER}/{PRIVATE_REPO_NAME}\", headers=invalid_headers)\n>       assert response.status_code == 404, \"Unauthorized access should fail (GitHub hides private repos)\"\nE       AssertionError: Unauthorized access should fail (GitHub hides private repos)\nE       assert 401 == 404\nE        +  where 401 = <Response [401]>.status_code\n\ntests\\api_tests\\test_authentication_api.py:58: AssertionError"}, "description": "Step 4: Verify that an invalid token cannot access a private repository.", "attachments": [{"name": "stdout", "source": "d8d0ecb1-b7c1-40f7-95a1-a93f38d3cd1c-attachment.txt", "type": "text/plain"}], "start": 1742175216307, "stop": 1742175216583, "uuid": "86bde9c3-e03e-4517-b32e-718c8595ac3f", "historyId": "4a08b46449c2e40b775d97367f5ce126", "testCaseId": "4a08b46449c2e40b775d97367f5ce126", "fullName": "tests.api_tests.test_authentication_api#test_private_repository_invalid_token", "labels": [{"name": "parentSuite", "value": "tests.api_tests"}, {"name": "suite", "value": "test_authentication_api"}, {"name": "host", "value": "BHUVANSPC"}, {"name": "thread", "value": "18552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_tests.test_authentication_api"}]}